### ISSUE TITLE
Table align option

### ISSUE THREAD

**Issue by sebkur** [MitchTalmadge/ASCIIGraph#13]
For the tables, it would be really useful to have an option on the `ASCIITable` class to set a column's justification, so that we can create tables with centered or right-aligned content. What do you think about it?

# Examples

Instead of this:

```
╔═══════════╤══════════╗
║ FirstName │ LastName ║
╠═══════════╪══════════╣
║ Foo       │ Bar      ║
╟───────────┼──────────╢
║ Kit       │ Kat      ║
╚═══════════╧══════════╝
``` 
I sometimes prefer this:
```
╔═══════════╤══════════╗
║ FirstName │ LastName ║
╠═══════════╪══════════╣
║ Foo       │      Bar ║
╟───────────┼──────────╢
║ Kit       │      Kat ║
╚═══════════╧══════════╝
``` 
Escpecially when numbers are involved:
```
╔═══════════╤══════════╗
║ Name      │ Score    ║
╠═══════════╪══════════╣
║ Foo       │        5 ║
╟───────────┼──────────╢
║ Kit       │      123 ║
╚═══════════╧══════════╝
```

**Comment by sebkur** [comment]
Partially implemented in #15

**Comment by MitchTalmadge** [comment]
Merged, will release 1.4.0 when I get a chance.
